src/index.js

```diff
@@ -0,0 +1,36 @@
+import './style.css';
+import pokeball from './assets/pokeball.png';
+import pokedex from './assets/Pokédex_logo.png';
+import { fetchPokemons } from './modules/pokemon.js';
+
+const $header = document.querySelector('.header');
+const $footer = document.querySelector('.footer');
+
+$header.innerHTML = `
+    <div class="img-container">
+        <div class="pokeball-container">
+            <img src="${pokeball}" alt="pokeball">
+        </div>
+        <div class="pokedex-container">
+            <img class="pokedex" src="${pokedex}" alt="Pokedex">
+        </div>
+    </div>
+    <nav class="list-links">
+        <a href="" id="Pokemons-category">Pokemons</a>
+        <a href="" id="About-category">About</a>
+        <a href="" id="Contact-category">Contact</a>
+    </nav>
+`;
```

Comment:
Great done the header looks good , but one critical element is missing which is the counter 

**project requirements :**
Counters We have counters in all the interfaces that show:
The number of items (home).
The number of comments (comments popup).